tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/appsettings.json

```diff
@@ -132,7 +132,8 @@
     "KeyVaultCertThumbPrint": "",
     "IsDevMode": true,
     "GraphUrl": "https://graph.microsoft.com",
-    "IsBackwardCompatible": true
+    "IsBackwardCompatible": true,
+    "IsClientMappedWithHierachy": false
```

Comment:
Added one more property in the general section of appsettings.json called "IsClientMappedWithHierachy". This will decide whether the taxonomy hierarchy needs to be filtered for client selectin or not